### PR TITLE
Add Cisco ISE format

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
Hi @squioc, you will find bellow the Cisco ISE format. Please can you review it and tell me if that logic is good or not ? and if there's a better way ?

### CODE REVIEW

File: Cisco/cisco-ise/tests/test_conf_events.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "dialect_uuid": "8a9894f8-d7bc-4c06-b96a-8808b3c6cade"
+      }
+    },
+    "message": "INFO: Configuration Changed: Admin=john.doe; Object Type=EPPurgeScheduler; Object Name=f36afcff-e3af-4a70-99c0-5e5304c1c336"
+  },
+  "expected": {
+    "message": "INFO: Configuration Changed: Admin=john.doe; Object Type=EPPurgeScheduler; Object Name=f36afcff-e3af-4a70-99c0-5e5304c1c336",
+    "event": {
+      "kind": "event",
+      "type": ["info"],
```
Comment (squioc) [inline Cisco/cisco-ise/tests/test_conf_events.json:15]: The ECS specification allows us defining finer [`event.type`](https://www.elastic.co/guide/en/ecs/8.6/ecs-allowed-values-event-category.html#ecs-event-category-configuration) for a configuration event.
```suggestion
      "type": ["change"],
```
Could you also add test for a  configuration creation

File: Cisco/cisco-ise/tests/test_netwoork_info_events.json
Comment (squioc) [inline Cisco/cisco-ise/tests/test_netwoork_info_events.json:1]: Typo in the name of this file: test_netwoork_info_events.json (the o is doubled in network)

File: Cisco/cisco-ise/tests/test_netwoork_warn_events.json
Comment (squioc) [inline Cisco/cisco-ise/tests/test_netwoork_warn_events.json:1]: Typo in the name of this file: test_netwoork_info_events.json (the o is doubled in network)

**Review by squioc — CHANGES_REQUESTED** [review]
Could you add more tests for network events and add test for miscellaneous events?

File: Cisco/cisco-ise/tests/test_network_info_events.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "type": ["info"],
+      "category": "network"
+    },
+    "observer": {
+      "vendor": "Cisco",
+      "product": "Cisco ISE"
+    },
+    "source": {
+      "domain": "servername",
+      "mac": "A4:57:0:64:47:C0:test1",
```
Comment (squioc) [inline Cisco/cisco-ise/tests/test_network_info_events.json:24]: This mac address is invalid. Only the first part `A4:57:0:64:47:C0` is the mac address, the second part `test1` seems to be the name of device

**Review by squioc — CHANGES_REQUESTED** [review]

**Review by squioc — COMMENTED** [review]
It seems that the test failed.

File: Cisco/cisco-ise/_meta/smart-descriptions.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      {
+        "field": "cisco.ise.config_action"
+      },
+      {
+        "field": "cisco.ise.config_object.name"
+      }
+    ]
+  },
+  {
+    "value": " Connection from {source.ip} to the server {source.domain} and this is the reason",
```
[the remark this answers is not in the document]
Comment (TOUFIKIzakarya) [reply Cisco/cisco-ise/_meta/smart-descriptions.json:14]: Oh yes, thank you for that Océane.

File: Cisco/cisco-ise/ingest/parser.yml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        filter: '{{parsed_kv_sentence.result.get("Error Message") != None and "Configuration" not in parsed_event.message.reason}}'
+      - set:
+          event.reason: '{{parsed_kv_sentence.result.get("Error Message")}}'
+        filter: '{{parsed_kv_sentence.result.get("Error Message") != None and "Configuration" not in parsed_event.message.reason}}'
+      - set:
+          event.reason: '{{parsed_kv_sentence.result.get("Failure Reason")}}'
+        filter: '{{parsed_kv_sentence.result.get("Failure Reason") != None and "Configuration" not in parsed_event.message.reason}}'
+      - set:
+          source.domain: '{{parsed_kv_sentence.result.get("Device Name")}}'
+        filter: '{{parsed_kv_sentence.result.get("Device Name") != None and "Configuration" not in parsed_event.message.reason}}'
```
[the remark this answers is not in the document]
Comment (TOUFIKIzakarya) [reply Cisco/cisco-ise/ingest/parser.yml:116]: Ok, so I need to modify it in all other filters. Thank you.

**Review by Adamowoc — APPROVED** [review]

File: Cisco/cisco-ise/tests/test_miscellaneous1.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco ISE",
+        "dialect_uuid": "8a9894f8-d7bc-4c06-b96a-8808b3c6cade"
+      }
+    },
+    "message": "2023-06-07 04:26:17.306 +0200 60198 INFO null: MnT purge event occurred, MESSAGE=completed successfully,"
```
Comment (Adamowoc) [inline Cisco/cisco-ise/tests/test_miscellaneous1.json:9]: Could you parse the "MESSAGE=completed successfully" and set [event.outcome](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-outcome.html) with it?

**Review by Adamowoc — CHANGES_REQUESTED** [review]

File: Cisco/cisco-ise/tests/test_miscellaneous1.json (hunk 5 above)
Comment (Adamowoc) [reply Cisco/cisco-ise/tests/test_miscellaneous1.json:9]: event.outcome doesn't seem to appear in the test, we'll see that later with @squioc

**Review by Adamowoc — APPROVED** [review]

File: Cisco/cisco-ise/_meta/manifest.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+uuid: 8a9894f8-d7bc-4c06-b96a-8808b3c6cade
+name: Cisco ISE
```
Comment (Adamowoc) [inline Cisco/cisco-ise/_meta/manifest.yml:2]:
```suggestion
name: Cisco ISE [BETA]
```

**Review by Adamowoc — APPROVED** [review]

**Review by squioc — APPROVED** [review]
LGTM

**Review by squioc — APPROVED** [review]

**Pull request merged**